[eas-cli] Add support for `submit:internal`

sjchmiela opened the pull request:
<!-- If this PR requires a changelog entry, add it by commenting the PR with the command `/changelog-entry [breaking-change|new-feature|bug-fix|chore] [message]`. -->
<!-- You can skip the changelog check by labeling the PR with "no changelog". -->

# Why

Please describe the motivation for this PR, and link to relevant GitHub issues, forums posts, or feature requests.

# How

How did you build this feature or fix this bug and why?

# Test Plan

Please describe how you tested this change and how a reviewer could reproduce your test, especially if this PR does not include automated tests! If possible, please also provide terminal output and/or screenshots demonstrating your test/reproduction.

szdziedzic reviewed (approved):
looks reasonable, one Q regarding commands directory structure:

sjchmiela commented:
Sorry, Newly Added Update Reviewers, I updated GraphQL types and had to adjust two tests (added `isPaused` and changed an assumed type of a temporary type fix). All is good!
